translations/strings.yaml

```diff
@@ -709,6 +727,15 @@ subscriptionManagement:
   addFirefoxRelay:
     value: Add Firefox Relay
     comment: Block of text appearing above a link to learn more about the VPN / Relay bundle offering.
+  valueProp1:
+    value: No logging of your network activity
+    comment: Value proposition used to upsell the product
```

Comment:
```suggestion
    comment: Value proposition used to promote the bundle with Relay
```

Reply:
"upsell" was probably a bad word choice here since this is unrelated to the relay bundle. Will replace with "promote"